### PR TITLE
Parallel building (frontend & backend) + unify running scripts

### PR DESCRIPTION
**Pull request by dragonsahead**
Docker build now runs in 4 layers:
- frontend (which builds now separately) // yarn build ...
- BE deps (lein deps...)
- Drivers (sh build_drivers...)
- Main builder (grabs everything from previous layers and builds the uberjar) (bin/build uberjar...)

it also unifies /app/run_metabase.sh between build and release containers

The argument behind this change is that the frontend takes a long time to build while the backend builder is sitting idle till the frontend finishes building. Therefore I tried to build the entire frontend separately while I also separated the drivers build from the final build (which was being done together). The results end up with a 15% reduction in the build time (original builds took 2500 secs on my computer with my internet connection and now take 1500 secs).

If we can get this to build driver by driver (each of them separately) it can be compressed even more, but you run out of RAM quickly as there is no way to contain the RAM consumption of the build.

### CODE REVIEW

**Review by ariya — CHANGES_REQUESTED** [review]
If this is tweaking the build, `yarn.lock` should not be changed at all (because then we're updating dependencies).

File: bin/docker/Dockerfile [hunk 1]
```diff
@@ -3,8 +3,12 @@ FROM adoptopenjdk/openjdk11:alpine-jre
 ENV FC_LANG en-US LC_CTYPE en_US.UTF-8
 
 # dependencies
+<<<<<<< HEAD
 RUN apk -U upgrade &&  \
     apk add --update --no-cache bash ttf-dejavu fontconfig curl java-cacerts && \
+=======
```
Comment (ariya) [inline bin/docker/Dockerfile:9]: Looks like this is a merge conflict?

**Comment by paoliniluis** [comment]
Sorry @ariya for the previous commit, it was a complete disaster. I force pushed one with the correct changes

**Review by ariya — APPROVED** [review]
LGTM

**Pull request merged**